schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY NOT NULL,
```

Comment:
- [ OPTIONAL] 
You can make the `id` column to `autoincrement` if you wish by adding `IDENTITY(1,1)`